src/layouts/Footer.tsx

```diff
@@ -28,7 +21,7 @@ const FooterView = () => (
       ]}
       copyright={
         <Fragment>
-          Copyright <Icon type='copyright' /> 2018 蚂蚁金服体验技术部出品
+          Copyright <Icon type='copyright' /> 2019 ...
```

Comment:
Suggestion here: Could replace with year calc operation.
```{{new Date().getFullYear()}}```

Reply:
Thanks for suggestion. Done.